### ISSUE TITLE
Missing <img> alt attribute should be an error, not a warning.

### ISSUE THREAD

**Issue by ckundo** [GoogleChrome/accessibility-developer-tools#10]
WebAIM indicates that [even presentational images should have alt attributes](http://webaim.org/techniques/images/alt_text#null), they should just be empty as an instruction to assistive tech to ignore them. Otherwise screen readers may read the file name or apply other heuristics to guess the description.

What do you think about elevating the severity on this audit to an error rather than a warning?

Here is the line: https://github.com/GoogleChrome/accessibility-developer-tools/blob/master/src/audits/ImageWithoutAltText.js#L27
